pkgs/development/tools/build-managers/tup/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, fetchFromGitHub, fuse, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  name = "tup-v${version}";
+  version = "0.7.3";
+
+  src = fetchFromGitHub {
+    owner = "gittup";
+    repo = "tup";
+    rev = "v${version}";
+    sha256 = "1x2grwmlf2izip4djb8cjwgl8p3x0bmfqwzjsc017mqi17qkijy8";
+  };
+
+  buildInputs = [ fuse pkgconfig ];
+
+  configurePhase = ''
+    sed -i 's/`git describe`/v${version}/g' Tupfile
+  '';
+
+  # Regular tup builds require fusermount to have suid, which nix cannot
```

Comment:
I'm still getting the hang of nix packaging, but doesn't including headers from and linking fuse mean that fuse must be in buildInputs?